temporal-normalizer/tools/TimeML-Normalizer/README.TXT

```diff
@@ -0,0 +1,41 @@
+====
```

Comment:
Yes, everything should be included. The evaluator is written in python, can that be included as a maven dependency?